libs-scala/test-evidence/tag/src/main/scala/com/daml/test/evidence/tag/Security.scala

```diff
@@ -106,6 +106,14 @@ object Security {
 
       /** Whether a secure configuration can effectively be enforced */
       case object SecureConfiguration extends Property
+
+      case object Confidentiality extends Property
+
+      case object Authentication extends Property
+
+      case object NonRepudiation extends Property
+
+      case object Resilience extends Property
```

Comment:
Could you please:
- Avoid duplicate properties (Authenticity vs Authentication, Privacy vs Confidentiality). To avoid migration effort, I would keep the old categories and add comment of the form `This is also known as Authentication.`.
- Add a ScalaDoc to new properties to explain what this is about. In particular for Resilience (against what?).

Reply:
What exactly do you mean by "Avoid duplicate properties (Authenticity vs Authentication, Privacy vs Confidentiality)" -- you want me to revise all existing `SecurityTest`s and replace Authenticity with Authentication, Privacy with Confidentiality?

Reply:
Remove either Authenticity or Authentication. Remove either Privacy or Confidentiality.

Up to you which one you drop. Feel free to drop the one that causes less migration effort.

Reply:
Got it. Done.

Reply:
> Remove either Authenticity or Authentication. Remove either Privacy or Confidentiality.

According to the diff, you haven't removed any of Authenticity, Authentication, Privacy, Confidentiality. So we end up with multiple properties for the same thing. This is exactly the situation I wanted to avoid in the first place.

To be frank, given this outcome I regret having pre-approved the PR.

Reply:
@cbley-da